tests/views_integration/Tribe/Events/Views/Modules/__snapshots__/Map_BasicTest__test_render_google_maps_iframe_with_mock_data__1.php

```diff
@@ -0,0 +1,9 @@
+<?php return '
+<iframe
+  title="Google maps iframe displaying the address to "
```

Comment:
This is not adding the venue title - we should ensure that it's passed or findable for the test.